File: 1_two_sum.rb
# @param {Integer[]} nums
# @param {Integer} target
# @return {Integer[]}
def two_sum(nums, target)
  nums.each_with_index do |num, i|
    j = nums.index(target - num)
    return [i, nums.index(target - num)] if j != i && !j.nil?
  end
end

File: 1_two_sum.py
class Solution(object):
    def two_sum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        for i, num in enumerate(nums):
            try:
                j = nums.index(target - num)
            except ValueError:
                j = None
            if (j != i) and (j is not None):
                return [i, j]
